### PR TITLE
Update developer guide with proposed sorting feature

### PR DESCRIPTION
**Pull request by Ashuh**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -132,6 +132,30 @@ The `viewImageWindow` is the displayed after it retrieves the set from `model`
...
+| `Phone`              | `phone`        |
+| `Email`              | `email`        |
+| `Favourite`          | `favourite`    |
+| `Address`            | `address`      |
+| `UserType`           | `usertype`     |
+| Number of `Property` | `num_property` |
+
+Sorting the list is done by using the `sort` command, which has the following syntax: `sort [KEYWORD]...`.
+
+If multiple attributes are specified, the first attribute is given the highest priority, while the last attribute is given the lowest priority. For example, `sort address name` will sort the list by `Address` first, followed by `Name` if `Address` is equal.
```
Comment (hsiaojietng) [inline docs/DeveloperGuide.md:153]: Maybe use another field rather than address, since we made it clear that every buyer/seller can have 1 address right?

**Review by hsiaojietng — COMMENTED** [review]
Just 1 matter that we might need to discuss

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (Ashuh) [reply docs/DeveloperGuide.md:153]: What this means is that if 2 people have the same `address`, they will be sorted according to `name` instead.

**Review by waffledood — APPROVED** [review]
LGTM! 👍

**Pull request merged**